### PR TITLE
xds: fix invalid comments

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: pilot/pkg/proxy/envoy/v2/ads.go [hunk 1]
```diff
@@ -391,7 +391,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 	go receiveThread(con, reqChannel, &receiveError)
 
 	for {
-		// Block until either a request is received or the ticker ticks
+		// Block until either a request is received or a push triggered.
```
Comment (ZackButcher) [inline pilot/pkg/proxy/envoy/v2/ads.go]: > a push **is** triggered

File: pilot/pkg/proxy/envoy/v2/ads.go [hunk 2]
```diff
@@ -177,7 +177,7 @@ type XdsConnection struct {
 
 	modelNode *model.Proxy
 
-	// Sending on this channel results in  push. We may also make it a channel of objects so
+	// Sending on this channel results in push. We may also make it a channel of objects so
```
Comment (ZackButcher) [inline pilot/pkg/proxy/envoy/v2/ads.go]: > in **a** push

File: pilot/pkg/proxy/envoy/v2/eds.go [hunk 3]
```diff
@@ -248,7 +248,7 @@ func (s *DiscoveryServer) StreamEndpoints(stream xdsapi.EndpointDiscoveryService
 	go receiveThread(con, reqChannel, &receiveError)
 
 	for {
-		// Block until either a request is received or the ticker ticks
+		// Block until either a request is received or a push triggered.
```
Comment (ZackButcher) [inline pilot/pkg/proxy/envoy/v2/eds.go]: > **is** triggered

File: pilot/pkg/proxy/envoy/v2/ads.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pilot/pkg/proxy/envoy/v2/ads.go]: done

File: pilot/pkg/proxy/envoy/v2/eds.go (hunk 3 above)
Comment (hzxuzhonghu) [reply pilot/pkg/proxy/envoy/v2/eds.go]: done

File: pilot/pkg/proxy/envoy/v2/ads.go (hunk 2 above)
Comment (hzxuzhonghu) [reply pilot/pkg/proxy/envoy/v2/ads.go]: done

**Pull request merged**